pkg/actors/actors_test.go

```diff
@@ -790,6 +791,46 @@ func TestOverrideTimerCancelsMultipleActiveTimers(t *testing.T) {
 	})
 }
 
+func timerRepeats(ctx context.Context, t *testing.T, repeats uint32) {
+	requestC := make(chan testRequest, 10)
+	appChannel := mockAppChannel{
+		requestC: requestC,
+	}
+	testActorsRuntime := newTestActorsRuntimeWithMock(&appChannel)
+	actorType, actorID := getTestActorTypeAndID()
+	fakeCallAndActivateActor(testActorsRuntime, actorType, actorID)
+
+	timer := createTimerData(actorID, actorType, "timer", "2s", "2s", repeats, "callback", "data")
+	err := testActorsRuntime.CreateTimer(ctx, &timer)
+	assert.Nil(t, err)
+
+	// timeout the test after 2s + repeats * 2s + 4s
+	timeout := time.Duration((repeats+3)*2) * time.Second
+
+	var cnt uint32
+L:
```

Comment:
It is worse than `LOOP` :(

Reply:
I liked it more.

Reply:
I kind of agree with @daixiang0. These "goto-like" syntax is not so readable. Let's avoid these, please.

Reply:
I cannot justify this ask. Please refer to my comments above: https://github.com/dapr/dapr/pull/3437#discussion_r671328143